### PR TITLE
Add section about using Azure CLI fro IoT Hub

### PR DESCRIPTION
**Pull request by ppapla**

### CODE REVIEW

File: docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Configure_Azure_IoT_Hub_integration.md [hunk 1]
```diff
@@ -47,3 +51,51 @@ Use the obtained credentials to establish the integration between {{ coiote_shor
...
+
+This section illustrates the configuration of Azure resources through the Command-Line Interface (CLI), offering a streamlined and scriptable approach to resource management. It is assumed that you have completed the following prerequisites:
+
+1. **Installed Azure CLI:**
+   Ensure that Azure CLI is installed on your system. If not, refer to the [official installation guide](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) for detailed instructions.
+
+2. **Authenticated to Azure CLI:**
+   Authenticate your Azure CLI by following the steps outlined in [this guide](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli-interactively).
+
+3. **Created Resource Group `coiote-dm-experiments`:**
```
Comment (Kuczu) [inline docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Configure_Azure_IoT_Hub_integration.md:66]: imo you should add a suggestion of the name, e.g. `Created Resource Group e.g. `coiote-dm-integration``, also I'm suggesting to change `coiote-dm-integration` to `coiote-dm-integration`

File: docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Configure_Azure_IoT_Hub_integration.md [hunk 2]
```diff
@@ -47,3 +51,51 @@ Use the obtained credentials to establish the integration between {{ coiote_shor
...
+az storage account show-connection-string -n ${USER}hub -g $GROUP
+
+# Create and configure an IoT hub
+az iot hub create -n $USER-hub -g $GROUP
+az iot hub connection-string show -n $USER-hub -g $GROUP
+```
+
+### Clean up resources
+
+To delete previously created Azure resources, including the specified storage account and IoT hub, execute this script:
```
Comment (Kuczu) [inline docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Configure_Azure_IoT_Hub_integration.md:93]: add here some kind of warning that after the deletion integration stop working

Comment (ppapla) [reply docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Configure_Azure_IoT_Hub_integration.md]: done

File: docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Configure_Azure_IoT_Hub_integration.md (hunk 1 above)
Comment (ppapla) [reply docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Configure_Azure_IoT_Hub_integration.md]: ok

**Review by Kuczu — APPROVED** [review]

**Pull request merged**